spec/helpers/quadicon_helper_spec.rb

```diff
@@ -15,18 +15,42 @@
   end
 
   describe "#render_quadicon_text" do
+    before(:each) do
+      @settings = {:display => {:quad_truncate => "m"}}
+    end
+
     subject { helper.render_quadicon_text(item, row) }
 
     let(:item) do
       FactoryGirl.create(:vm_vmware)
     end
 
     let(:row) do
-      Ruport::Data::Record.new(:id => 10000000000534)
+      Ruport::Data::Record.new(:id => 10_000_000_000_534, "name" => name)
```

Comment:
This may be in the original, so don't change if so, but

1. We shouldn't be underscorizing ids (yes, even if Rubocop says so)
2. We shouldn't be hardcoding ids at all....curious why this is needed.  Part of the reason to not hardcode ids is that it conflicts with proper region values.